### PR TITLE
Fix pyproject.toml tool.mypy.overrides parsing

### LINKED ISSUE #145: [tool.mypy.overrides] sections are dropped from config
Hello everyone,

I noticed that when using `pyproject.toml` to configure mypy, the pytest-mypy-plugins plugin silently drops overrides sections:

```toml
[[tool.mypy.overrides]]
module = "mymodule"
ignore_missing_imports = true
```

I have a PR ready to fix this, and will push it right away.

Cheers,
Andreas 😃

### PR DESCRIPTION
**Pull request by ap--**
Closes #145

### CODE REVIEW

File: pytest_mypy_plugins/configs.py [hunk 1]
```diff
@@ -54,6 +55,19 @@ def join_toml_configs(
...
+
+        # construct toml output
+        min_toml = tomlkit.document()
+        min_tool = tomlkit.table(is_super_table=True)
+        min_toml.append("tool", min_tool)
+        min_tool.append("mypy", tool_mypy)
+
+        # strip leading comments
+        for i, item in enumerate(min_toml.body):
+            if not isinstance(item, (Comment, Whitespace)):
```
Comment (sobolevn) [inline pytest_mypy_plugins/configs.py:68]: Let's add a leading comment to test that line 👍

File: pytest_mypy_plugins/configs.py [hunk 2]
```diff
@@ -54,6 +55,19 @@ def join_toml_configs(
...
+                break
+        min_toml.body[:] = min_toml.body[i:]
```
Comment (sobolevn) [inline pytest_mypy_plugins/configs.py:70]:
```suggestion
        min_toml.body = min_toml.body[i:].copy()
```

This seems to be the same thing, but easier to read?

**Review by sobolevn — COMMENTED** [review]
Thanks! Two minor comments 👍

File: pytest_mypy_plugins/configs.py (hunk 1 above)
Comment (ap--) [reply pytest_mypy_plugins/configs.py:68]: All three pyproject.toml test files have leading comments. I had to add this to ensure the tests for pyproject1.toml and pyproject2.toml pass:

https://github.com/typeddjango/pytest-mypy-plugins/blob/a92e6bcf5a7c2057498df93ea3105b4b11886452/pytest_mypy_plugins/tests/test_configs/pyproject1.toml#L1-L2

https://github.com/typeddjango/pytest-mypy-plugins/blob/a92e6bcf5a7c2057498df93ea3105b4b11886452/pytest_mypy_plugins/tests/test_configs/pyproject2.toml#L1

File: pytest_mypy_plugins/configs.py (hunk 2 above)
Comment (ap--) [reply pytest_mypy_plugins/configs.py:70]: `tomlkit.item.Container.body` is read-only:

https://github.com/python-poetry/tomlkit/blob/911cccd630965ff423316e25b4685ecf7df0ec0a/tomlkit/container.py#L42-L44

So this is the only way to modify in place.

**Comment by ap--** [comment]
Thanks for the fast response!

File: pytest_mypy_plugins/configs.py (hunk 1 above)
Comment (ap--) [reply pytest_mypy_plugins/configs.py]: Nevermind.
My local installation was broken. This code path was not required and never used.

I removed it.

File: pytest_mypy_plugins/configs.py (hunk 2 above)
Comment (ap--) [reply pytest_mypy_plugins/configs.py]: Nevermind.
My local installation was broken. This code path was not required and never used.

I removed it.

**Review by sobolevn — APPROVED** [review]
Thank you!

**Pull request merged**